preprocessing/specification.md

```diff
@@ -91,7 +91,8 @@ In the NDJSON, each row contains a sequence entry version and a list of errors a
         alignedNucleotideSequences,
         nucleotideInsertions,
         alignedAminoAcidSequences,
-        aminoAcidInsertions
+        aminoAcidInsertions,
+        files
```

Comment:
I know that's not much, but rather than writing more here, I think that we should first add general documentation about how to upload files via the API and link to that.